pkg/registry/core/serviceaccount/strategy.go

```diff
@@ -82,3 +89,10 @@ func (strategy) WarningsOnUpdate(ctx context.Context, obj, old runtime.Object) [
 func (strategy) AllowUnconditionalUpdate() bool {
 	return true
 }
+
+func warnIfHasEnforceMountableSecretsAnnotation(ctx context.Context, serviceAccount *api.ServiceAccount) {
+	_, ok := serviceAccount.Annotations[sa.EnforceMountableSecretsAnnotation]
+	if ok {
+		warning.AddWarning(ctx, "", fmt.Sprintf("deprecated annotation %q on service account %s/%s. %s", sa.EnforceMountableSecretsAnnotation, serviceAccount.Namespace, serviceAccount.Name, sa.WarningMessage))
```

Comment:
Just pointing out something I saw in the code docs:

```go
// Be brief; limit warnings to 120 characters if possible.
```

I am not sure how to limit ourselves to such a small character count while still being useful, so IMO we should leave it as-is.